tests/unit/test-loader-test.js

```diff
@@ -3,7 +3,11 @@ import { module, test } from 'qunit';
 
 module('Unit | test loader', {
   beforeEach() {
-    this.TestLoader = window.require('ember-cli/test-loader')['default'];
+    try {
+      this.TestLoader = window.require('ember-cli-test-loader/test-support/index')['default'];
```

Comment:
it is also somewhat strange that this is coupled to `window` at all, isn't just ambient require sufficient?